### DISCUSSION TITLE
Welcome to Car_Showroom_Management_System Discussions!

### DISCUSSION THREAD

**Discussion by Anurag-Negi28** [Anurag-Negi28/Car_Showroom_Management_System#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Anurag-Negi28** [comment]
Hey guys all the discussion can be done here related to the Car Showroom Management System.

**Reply by HardikYadav99** [reply]
Hi everyone, I have made a menu bar on the website
Tell me if some changes has to be done in the menu box.
Also suggest if you have better options.
<img width="1470" alt="github" src="https://github.com/Anurag-Negi28/Car_Showroom_Management_System/assets/143898154/a7d59f57-cec1-41ae-a5ef-ae4fc488915e">

**Reply by Anurag-Negi28** [reply]
A dashboard can also be created in the menu bar having showroom name in middle.

**Reply by HardikYadav99** [reply]
Yeah Anurag, I have planned the same I will be working on it.

**Comment by madhvendra13** [comment]
Hi Anurag, what features should we plan for our management application ?

**Reply by Anurag-Negi28** [reply]
I think we can add a login page for the manager of the car showroom and employee for authentication

**Comment by Anurag-Negi28** [comment]
we should do a meet tomorrow at 2:oo pm regarding the work distribution for the project.

**Reply by Anurag-Negi28** [reply]
Today's Meeting outcomes - Work distribution
Frontend part - @HardikYadav99
About Section - @Anshsaxenaa
Login page and readme - @vrindagupta25
Backend part - @Anurag-Negi28 
Dark Mode implementation - @Piyush107193
 Car Data Collection - @madhvendra13 @HardikYadav99

**Comment by HardikYadav99** [comment]
We need a name for our showroom, everybody feel free to suggest a wonderful name.

**Reply by Anurag-Negi28** [reply]
How about Deals on Wheels?

**Reply by Anurag-Negi28** [reply]
We should also add a tagline

**Reply by HardikYadav99** [reply]
Wheels on deals is a good name. 
Tag line is yet to be finalised.

**Comment by HardikYadav99** [comment]
Hi Everyone, 
I have uploaded the work I have done on the website till now. 
Acknowledge it once.Let me know if any changes has to be done on the first page of the website.
Note- Work is still going on the photos which are below the video.

**Reply by Anurag-Negi28** [reply]
I had a suggestion that in the menu bar can you add this feature -clicking on the menu button will close the menu bar
![image](https://github.com/Anurag-Negi28/Car_Showroom_Management_System/assets/123005508/725659f3-dd04-402b-be31-edb7cdd67d00)

**Reply by HardikYadav99** [reply]
I will be working on it and I will let you know how this will take place.

**Reply by Anurag-Negi28** [reply]
ok great

**Comment by madhvendra13** [comment]
logo added . pls review

**Reply by madhvendra13** [reply]
How about adding animation?

**Reply by Anurag-Negi28** [reply]
I think it is not required as the video is already running

**Reply by HardikYadav99** [reply]
I think there should name in the logo of our showroom. Logo should more eye-catching, this logo more looks like car repairing rather than the car showroom.

**Reply by HardikYadav99** [reply]
<img width="309" alt="Screenshot 2023-11-04 at 12 17 23 AM" src="https://github.com/Anurag-Negi28/Car_Showroom_Management_System/assets/143898154/ecc9c238-8ec2-42d7-81a9-ff379bb22165">
I would suggest this type of single and elegant company logo, I think basic text with a good font and color will get our work done more smoothly.

**Reply by Anurag-Negi28** [reply]
Works well
anyone who would like to volunteer for designing the name of the showroom - Deals on Wheels

**Comment by vrindagupta25** [comment]
Everything’s going great till now let’s keep up the spirit

**Reply by Anurag-Negi28** [reply]
yeah sure

**Comment by Anurag-Negi28** [comment]
I have pushed the use case you can check it here-
![Use_case](https://github.com/Anurag-Negi28/Car_Showroom_Management_System/assets/123005508/de1906b0-5e90-4a9a-bcf4-85d15a700f1d)

**Reply by Anurag-Negi28** [reply]
I will also be uploading the tech stack that we will be using in the management system

**Reply by Anurag-Negi28** [reply]
Here is the tech stack
![Tech_stack](https://github.com/Anurag-Negi28/Car_Showroom_Management_System/assets/123005508/d2730012-a63b-4549-b845-d81090b11162)

**Reply by HardikYadav99** [reply]
okay, front end is clear to me, I will plan and do things accordingly.

**Comment by Anurag-Negi28** [comment]
I have added a hover feature in the style.css file with a scale of 1 and time 1 second
![image](https://github.com/Anurag-Negi28/Car_Showroom_Management_System/assets/123005508/c217c661-b287-403c-949a-4e298f28418b)
You can provide your reviews

**Reply by HardikYadav99** [reply]
Thats look more decent we will continue with this.

**Comment by vrindagupta25** [comment]
car names:
Sedans:
LuxoMax S4
EleganceDrive E7
PrestigeX P6

sports Cars:
Speedster S2
TrackMaster T7
VelocityV8 V12

Luxury Cars:
OpulentOasis O4
RoyalElegance R8
PlatinumPrestige P5

Hybrid Cars:

EcoCharge E4
GreenFusion G2
HybridHarmony H9 

please shortlist among the given car models

**Reply by HardikYadav99** [reply]
I have sorted the car names and added on bunch of cars.

**Comment by HardikYadav99** [comment]
<img width="1470" alt="employee data" src="https://github.com/Anurag-Negi28/Car_Showroom_Management_System.github.io/assets/143898154/6f01e7ea-b2ed-47f1-b0af-64a00a445ef5">
Here is the Employee Data Page for the car showroom, I have added the 4 employee with their respective information. Do let me know if I have to make some changes, open for suggestions.

**Comment by HardikYadav99** [comment]
I have deployed all the codes and photos that are utilised in Employee Data. You can check it in Employee Data file under Source Code.Let me know if any changes has to be done.

**Comment by Anurag-Negi28** [comment]
Hey guys now I have started working on the backend part in django, so you can upload the html pages that are left.

**Comment by vrindagupta25** [comment]
I have made the login page

**Comment by vrindagupta25** [comment]
I have also made an about page in which I have added 6 employee names which are our team member names and I request everone to send their profile pictures I will be adding it on our about page

**Comment by Anurag-Negi28** [comment]
@HardikYadav99 can you provide the car details that you have added like car name, description, brand, milage, price .I need them to add in the database.
